.github/workflows/kuksa_databroker-cli_build.yml

```diff
@@ -27,7 +27,7 @@ on:
   workflow_dispatch:
 
 concurrency:
-      group: ${{ github.ref }}-${{ github.workflow }}
+      group: ${{ github.ref }}-${{ github.workflow }}-dbcli
```

Comment:
do we wanna call this databroker-cli as well? short version my cause confusions.